LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/LoRaDevice.cs

```diff
@@ -322,7 +322,12 @@ public virtual async Task<bool> InitializeAsync(NetworkServerConfiguration confi
                     NetID = twin.Properties.Reported[TwinProperty.NetID].Value as string;
 
                 if (twin.Properties.Reported.Contains(TwinProperty.DevNonce))
-                    DevNonce = twin.Properties.Reported[TwinProperty.DevNonce].Value as string;
+                {
+                    var devNonceString = twin.Properties.Reported[TwinProperty.DevNonce].Value as string;
+                    var buffer = new byte[LoRaWan.DevNonce.Size];
+                    _ = Hexadecimal.TryParse(devNonceString, buffer);
```

Comment:
Same comment as earlier. I think an overload of `Hexadecimal.TryParse` for `ushort` will help here too.

Reply:
Addressed with 683e769799ae8a952bd44c48407d86ee7ec751a9